### PR TITLE
Adding control-border color to the theme palette and applying it to the form controls

### PR DESCRIPTION
**Pull request by ollips**
Task: task-568516

Link: preview-364

[Explain the context of this pull request here]

## Testing

[Input](https://design.docs.microsoft.com/pulls/364/components/input.html)
[Select](https://design.docs.microsoft.com/pulls/364/components/select.html)
[Textarea](https://design.docs.microsoft.com/pulls/364/components/textarea.html)
[Checkbox](https://design.docs.microsoft.com/pulls/364/components/checkbox.html)

1. Review the pages linked above. Controls border color should look slightly lighter and match the design. Check on different themes

### CODE REVIEW

File: .changeset/sharp-planets-dress.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': patch
+---
+
+Adding control-theme to the theme palette and applying it to the form controls; fixing select component font size.
```
Comment (wibjorn) [inline .changeset/sharp-planets-dress.md:5]: color-border, not control-theme

Comment (ollips) [reply .changeset/sharp-planets-dress.md:5]: 🤦🏻‍♀️

Comment (wibjorn) [reply .changeset/sharp-planets-dress.md:5]: Adding control-border to the theme palette and apply it to the form controls; fix select component font size.

Comment (wibjorn) [reply .changeset/sharp-planets-dress.md:5]: Different changeset: " fix select component font size."

Comment (ollips) [reply .changeset/sharp-planets-dress.md]: Oh, I didn't know I can do it several times in the same PR. I'll update

Comment (ollips) [reply .changeset/sharp-planets-dress.md]: Updated

**Review by wibjorn — APPROVED** [review]

**Review by jdanyow — APPROVED** [review]

**Pull request merged**